rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/Sw360ProjectService.java

```diff
@@ -566,4 +567,10 @@ public int countProjectsByReleaseIds(Set<String> releaseIds) {
             return 0;
         }
     }
+
+    public AddDocumentRequestSummary createClearingRequest(ClearingRequest clearingRequest, User sw360User) throws TException {
+        ProjectService.Iface sw360ProjectClient = getThriftProjectClient();
+        return sw360ProjectClient.createClearingRequest(clearingRequest, sw360User, " ");
```

Comment:
if you pass empty project url, the email notification will not have link to the project.

Reply:
I cannot provide the link in the email triggered via rest api as there is no ResourceRequest here to create a LiferayPortletURL, also we are moving out of liferay so it would not make sense to include it. As an alternate, I have now provided projectid instead of empty string:
![image](https://github.com/eclipse-sw360/sw360/assets/115608771/dd997414-1a30-42b1-95e4-34b241d13edf)